CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ If you are in doubt, feel free to submit it and we'll have a look.
 
 To be considered popular enough to be within our scope, a brand must be in existence for at least one year (from date of first stable release, where applicable) and meet one of the following metrics of popularity, in order of preference:
 
-1. Its website's Similarweb global rank is in the top 500k.
+1. Its website's [Similarweb global rank](https://www.similarweb.com/top-websites/) is in the top 500k.
```

Comment:
This link does not tell the global rank. Would you mind removing the link?

Reply:
Is this the correct link? https://www.similarweb.com/website/ maybe just change?

Reply:
The link is unneeded here.

Reply:
Ah ok, should be okay now.